xbmc/windowing/WinEventsLinux.h

```diff
@@ -19,27 +19,23 @@
  */
 
 #pragma once
+
```

Comment:
cut the dependency to windowing. read input events by a dedicated thread (like lirc does) and send to CApp's inbound port.